### PR TITLE
Updated docs for importing commitments

### PR DESCRIPTION
**Pull request by shuldt**
## Changes in this PR

This PR updates the documentation for importing commitments to reflect the current state of the repo.

### CODE REVIEW

File: doc/import-commitments.md [hunk 1]
```diff
@@ -17,13 +17,15 @@ RODA-ID,100000,2023-01-02
...
+The import script does not accept 'Commitment value' in the form of a string. If the csv has any values provided as strings, the string quotes around the number must be removed before running.
+
 A RODA user email address to be used to log the user who is recorded in the
 history log as importing the commitments.
 
 No authorisation is performed, as long as the RODA identifier exists and does
 not already have a commitment, the importer will set one.
 
-Copy the csv to production, [see details here](./importing-new-delivery-partner-data.md#import-activity-data-in-production)
+Copy the csv to production, [see details here](./uploading-and-dowloading-files.md)
```
Comment (Gweaton) [inline doc/import-commitments.md:28]: typo in `dow[n]loading-files` in the link here.

File: doc/import-commitments.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ At this point the application can only import commitments.
 ## Importing
 Commitments are imported by the `commitments:import` rake task.
 
-You will need a csv of valid commitments provided by BEIS:
+You will need a csv of valid commitments provided by BEIS. 'RODA indentifier' and 'Commitment value' are mandatory. 'Transaction date' is optional.
```
Comment (Gweaton) [inline doc/import-commitments.md:11]: I wonder if we should remove the `Transaction date` from this completely, as it looks like it's not actually being used here - I don't know why I added it to this documentation 🤦

File: doc/import-commitments.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ At this point the application can only import commitments.
...
 
 ```csv
 RODA identifier,Commitment value,Transaction date
```
Comment (Gweaton) [inline doc/import-commitments.md:14]: Let's remove `Transaction date` from here too

File: doc/import-commitments.md (hunk 2 above)
Comment (Gweaton) [reply doc/import-commitments.md:11]: also, small typo in `RODA i[n]dentifier` here.

**Review by Gweaton — COMMENTED** [review]
Thanks for updating this - would be good to add a couple of small tweaks and tidy up a couple of typos

File: doc/import-commitments.md (hunk 1 above)
Comment (yndajas) [reply doc/import-commitments.md:28]: The i[n]dentifier stole it

**Review by Gweaton — APPROVED** [review]

File: doc/import-commitments.md [hunk 4]
```diff
@@ -11,19 +11,21 @@ Commitments are imported by the `commitments:import` rake task.
 You will need a csv of valid commitments provided by BEIS:
 
 ```csv
-RODA identifier,Commitment value,Transaction date
+RODA identifier,Commitment value
 RODA-ID,100000,2023-01-02
```
[the remark this answers is not in the document]
Comment (shuldt) [reply doc/import-commitments.md:15]: Bah, missed that. thanks!

**Pull request merged**